prac_08/box_layout_demo.py

```diff
@@ -0,0 +1,20 @@
+from kivy.app import App
+from kivy.lang import Builder
+
+
+class BoxLayoutDemo(App):
+    def build(self):
```

Comment:
As far as i know we should add docstrings to every class / method in a .py file
If you check the examples from prac 8 provided by lindsay. Mostly they have docstrings for App classes.